### ISSUE TITLE
Clicking on the an arrow of a Hideable Sections does not interact with it

### ISSUE THREAD

**Issue by coastalwhite** [trimental/inlyne#82]
Clicking on the arrow of a hideable section does not interact with that section. Instead, you can click on the title text to interact.

The arrow should also cause interaction.

**Comment by trimental** [comment]
Yes this definitely makes sense to me. I'll try to implement this soon. Thanks for raising this issue 👍.
